cypress/integration/learn/challenges/output.js

```diff
@@ -40,7 +40,7 @@ describe('Classic challenge', function() {
   it('shows test output when the tests are run', () => {
     cy.visit(locations.index);
     // first wait for the editor to load
-    cy.get(selectors.editor, { timeout: 10000 });
+    cy.get(selectors.editor, { timeout: 15000 });
```

Comment:
Is Firefox particularly slow at this?

Reply:
I'm guessing so. Since this is the only test that was randomly failing in Firefox, I'd say it's a good guess.

Reply:
Right, thanks for confirming.  I'll keep that in mind, but we don't need to worry about it right now.